EIPS/eip-####.md

```diff
@@ -0,0 +1,97 @@
+---
```

Comment:
```suggestion
---
eip: 7797
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.